bitbots_buttons:
  ros__parameters:
    manual_penalty: True
    speak_active: True
    short_time: 2.0 #[s]
    debounce_time: 0.1 #[s]
    speak: true
